entries/repo-website-overview.puml

```diff
@@ -0,0 +1,80 @@
+@startuml Repos-Sites
+
+  package .github as GIT {
+      diamond dia1
+  }
+
+  package data as DAT {
+      diamond dia2
+  }
+
+  package theme as THE {
+      diamond dia3
+  }
+
+  package community as COM {
+      object "Community.Seedcase-Project.org" as CSP {
+        Project blocks
+        Roadmap
+        Guides
+        Posts
+        Outreach
+        Code of Conduct
```

Comment:
It might be overkill to have all the sections written out here (same for the rest of the repos)? I'm thinking that it will be thing that's easy to forget if we change something in the repos? 
On the other hand, it does give an overview of the content in each of them... Not sure what my opinion is here.

Reply:
Same here, I went with the full option in the end, just so that we have an idea of what is inside.

Reply:
@lwjohnst86 Thoughts?